thorvg/examples/thorvg-example/main/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+dependencies:
+  espressif/esp32_p4_function_ev_board:
```

Comment:
can we use esp32_p4_function_ev_board_noglib instead? because this example doesn't need a LVGL port